README.md

```diff
@@ -51,6 +51,14 @@ core distribution of the Collector. Typically, these contributions are vendor
 specific receivers/exporters and/or components that are only
 useful to a relatively small number of users.
 
+## Creating your own OpenTracing Collector distribution
```

Comment:
Are you sure this is `OpenTracing`?

Reply:
OMG. Great catch! That just reflects that I keep saying "OpenTracing" when I actually mean "OpenTelemetry"...